core/storage/email/gae_models_test.py

```diff
@@ -2,14 +2,14 @@
 #
 # Copyright 2014 The Oppia Authors. All Rights Reserved.
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
+# Licensed under the Apache License, Version 2.0 (the 'License');
```

Comment:
Why is this changed to single quotes? The pattern across files is to use double quotes.